libra-node/src/lib.rs

```diff
@@ -220,6 +220,46 @@ fn setup_debug_interface(config: &NodeConfig, logger: Option<Arc<Logger>>) -> No
     NodeDebugService::new(addr, logger)
 }
 
+async fn periodic_state_dumb(node_config: NodeConfig, db: DbReaderWriter) {
```

Comment:
dump?

Maybe just call it periodic_state :)

Reply:
dang i messed up the actual function name too?